### PR TITLE
Handle gently no supported clusters scenario

### PR DESCRIPTION
**Pull request by wsowa**
Signed-off-by: Witold Sowa <witold.sowa@gmail.com>

### CODE REVIEW

**Comment by wsowa** [comment]
Currently an NPE is thrown from `getExpectedUpdatePeriod` due to `min()` on empty collection in scenario where none of the supported clusters is found on the device. I got to this when playing with a simple binary sensor device (supports binary input cluster). Support for that cluster is in separate PR

File: org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java [hunk 1]
```diff
@@ -425,10 +425,16 @@ private synchronized void doNodeInitialisation() {
...
 
-        int expectedUpdatePeriod = getExpectedUpdatePeriod(channels);
-        expectedUpdatePeriod = (expectedUpdatePeriod * POLLING_OR_REPORTING_FACTOR) + POLLING_OR_REPORTING_MARGIN;
-        logger.debug("Setting ONLINE/OFFLINE timeout interval to: {}", expectedUpdatePeriod);
-        isAliveTracker.addHandler(this, expectedUpdatePeriod);
+        if (channels.isEmpty()) {
+            logger.warn("{}: No supported clusters found", nodeIeeeAddress);
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_INITIALIZING_ERROR, "No supported clusters found");
+            return;
+        } else {
```
Comment (cdjackson) [inline org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java]: Can I suggest to remove the ```else``` statement.

Comment (wsowa) [reply org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/handler/ZigBeeThingHandler.java]: done

**Comment by cdjackson** [comment]
Thanks. I'll merge once CI completes.

**Pull request merged**